libmiemu/controller.h

```diff
@@ -1,8 +1,35 @@
 #ifndef CONTROLLER_H
 #define CONTROLLER_H
 
+#include "datapath/signals.h"
+
+#define ADDI 0000
```

Comment:
If you want to `#define` these, do it in the implementation file so you don't pollute the namespace of whoever includes controller.h.